content/docs/reference/config/installation-options/index_version_diff.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Installation Options - Generate differences with previous version
+description: Describes the differences between the current and previous version of installation options from the Helm charts.
+weight: 30
+keywords: [kubernetes,helm]
+---
+
+## `ingress` options
+
+| Key (Old Key ----> New Key) | Default Value (Old Value ----> New Value) | Description (Old Description ---> New Description)|
+| --- | --- | --- |
+| `ingress.service.ports.name` | `https -----> http` | `` |
+
+## `global` options
+
+| Key (Old Key ----> New Key) | Default Value (Old Value ----> New Value) | Description (Old Description ---> New Description)|
```

Comment:
how bout breaking this table into
`| Old Key | New Key | Old Default | New Default | Old Description | New Description |`